### DISCUSSION TITLE
BasicNft: Lesson-10: Foundry-devops is not having the bash script

### DISCUSSION THREAD

**Discussion by ShabihEthSec** [Cyfrin/foundry-full-course-cu#3042] [Q&A]
Hello there! I am installing the `foundry-devops` tool today and there is no `get-recent-deployment.sh` bash script in the `src` folder. What should I do?
![{40C4BC17-102E-4BC1-96F7-E812E40CB2DF}](https://github.com/user-attachments/assets/47ef1737-81c6-45ec-8b39-dc5633e3d1cb)

**Comment by EngrPips** [comment]
Hello @gitShabih, `get-recent-deployment` is not a stand-alone file anymore but a function inside of the `DevOpsTools.sol` contract, so you can still access the `get-recent-deployment` through the `DevOpsTools.sol`. Please check this and confirm.
